docs/csharp/language-reference/builtin-types/reference-types.md

```diff
@@ -80,7 +80,61 @@ for (int i = 0; i < str.Length; i++)
 // Output: t e s t
 ```
 
-String literals are of type `string` and can be written in two forms, quoted and `@`-quoted. Quoted string literals are enclosed in double quotation marks ("):
+### String literals
+
+String literals are of type `string` and can be written in three forms, raw, quoted, and `@`-quoted. Raw string literals are enclosed in a minimum of three double quotation marks ("""):
+
+```csharp
+"""
+This is a multi-line
+    string literal with the second line indented.
+"""
+```
+
+*Raw string literals* are available beginning in C# 11. Raw string literals can contain arbitrary text without requiring escape sequences. Raw string literals can include whitespace and new lines, embedded quotes, and other special characters. You can even include a sequence of three (or more) double quote characters. If your text requires an embedded sequence of quotes, you start and end the raw string literal with more quote marks, as needed:
+
+```csharp
+"""""
+This raw string literal has four """", count them: """" Four! 
+embedded quote characters in a sequence. That's why it starts and ends
+with five double quotes.
+
+You could extend this example with as many embedded quotes as needed for your text.
+"""""
+```
+
+Raw string literals typically have the starting and ending quote sequences on separate lines from the embedded text. Multiline raw string literals supports strings that are themselves quoted strings:
+
+```csharp
+var message = """
+"This is a very important message."
+""";
+Console.WriteLine(message);
+// output: "This is a very important message."
+```
+
+The closing quote sequence dictates the leftmost column for the string literal. You can indent a raw string literal to match the overall code format:
+
+```csharp
+var message = """
+    "This is a very important message."
+    """;
+Console.WriteLine(message);
+// output: "This is a very important message."
+// The leftmost whitespace is not part of the raw string literal
+```
+
+Columns to the right of the ending quote sequence are preserved. This enabled raw strings for data formats such as JSON, YAML, or XML. The compiler issues an error if any of the text lines extend to the left of the closing quote sequence.
```

Comment:
"this enabled"?

Reply:
consider an example with multiple lines.  maybe 3 lines like so:

```c#
var json= """
    {
        "prop": 0
    } 
    """;
```

Showing that the lines with `{` and `}` are not indented, but the middle line is.